python/across_api/across/user.py

```diff
@@ -0,0 +1,250 @@
+from secrets import token_urlsafe
```

Comment:
I think that most of this file will not survive the transition to OIDC.

Reply:
Removal of auth code will resolve this.

Reply:
Fixed by cdf0e9f